TeamCharles/initial-site
# 15-view-by-type :: View Product Listings by Product Type

## Status

**READY**
## Documentation

[X] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

NO
## Description

Created a view for products of a particular Product Type.
## Related PRs

This builds on top of PR #68 
## Resolves Issue Number

Resolves #15 
## Deploy Notes

N/A
## Steps to Test or Reproduce
- Navigate to localhost:5000
- Click on any product type
- Verify that all products of that type are listed

``` sh
git pull --prune
git checkout 15-view-by-type
dotnet run
```
## Impacted Areas in Application

List general components of the application that this PR will affect:
- Product view
